projects/aca-content/src/lib/services/content-management.service.spec.ts

```diff
@@ -604,9 +634,9 @@ describe('ContentManagementService', () => {
       subject = new Subject<string>();
       spyOn(nodeActions, 'moveNodes').and.returnValue(subject);
 
-      spyOn(notificationService, 'openSnackMessageAction').and.returnValue({
-        onAction: () => of(null)
-      } as MatSnackBarRef<SimpleSnackBar>);
+      openSnackMessageActionSpy.and.returnValue({
+        onAction: () => of(new Subject<void>())
+      } as unknown as MatSnackBarRef<SimpleSnackBar>);
```

Comment:
You don't need unknown. I guess you had issue because there is assigned incorrect value to onAction above. If you check what's type of onAction result in MatSnackBarRef then it expects ```Observable<void>``` but you assign ``` of(new Subject<void>()) ``` which is ```Observable<Subject<void>>``` type so types are incorrect. You can use following instead:

```ts
openSnackMessageActionSpy.and.returnValue({
        onAction: () => of(undefined)
      } as MatSnackBarRef<SimpleSnackBar>);